requirements.txt

```diff
@@ -4,17 +4,18 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
-amqp==1.4.9               # via kombu
+-e git+https://github.com/steve-gregory/billiard.git#egg=billiard
```

Comment:
We are not maintaining this. And if I have to put this into our repo, then everything in the `requirements.txt` and `dev_requirement.txt` probably should be in our repo?